src/pretalx/agenda/views/talk.py

```diff
@@ -124,8 +128,14 @@ def get_context_data(self, **kwargs):
         context['recording_iframe'] = self.recording.get('iframe')
         context['speakers'] = []
         for speaker in slot.submission.speakers.all():
-            speaker.talk_profile = speaker.event_profile(event=self.request.event)
-            speaker.other_talks = event_talks.filter(submission__speakers__in=[speaker])
+            speaker.talk_profile = speaker.event_profile(
+                event=self.request.event)
```

Comment:
Please revert changes like this one, as they don't add anything and don't follow the style guidelines of the project.